src/components/Trade/hooks/useAccountsService.tsx

```diff
@@ -66,18 +62,9 @@ export const useAccountsService = () => {
   )
 
   // Set buyAssetAccountId
-  useEffect(() => {
-    setValue(
-      'buyAssetAccountId',
-      // If the swapper does not support cross-account trades the buyAssetAccountId must match the sellAssetAccountId
-      swapperSupportsCrossAccountTrade ? buyAssetAccountId : sellAssetAccountId,
```

Comment:
This was part of the incorrect receive account problem, and would occur when, for whatever reason, we did not get a swapper in the first render.